damus/Views/Relays/RelayPicView.swift

```diff
@@ -42,19 +42,24 @@ struct InnerRelayPicView: View {
             Color(uiColor: .secondarySystemBackground)
 
             if let url {
-                KFAnimatedImage(url)
-                    .imageContext(.pfp, disable_animation: disable_animation)
-                    .onFailure { _ in
-                        failedImage = true
-                    }
-                    .cancelOnDisappear(true)
-                    .configure { view in
-                        view.framePreloadCount = 3
-                    }
-                    .placeholder { _ in
-                        Placeholder(url: url)
-                    }
-                    .scaledToFit()
+                if url.absoluteString.contains(".onion") {
+                   Image("tor")
+                        .scaledToFit()
+                } else {
```

Comment:
Should we add the Tor icon in addition to the relay image (like a badge, for example), instead of replacing the relay icon?

Reply:
Yes, this makes more sense as relays have a unique icon. This information serves to more quickly communicate which relays a user is subscribed to.